javascript-loops-practice/filter-out-strings.js

```diff
@@ -1,9 +1,10 @@
 /* exported filterOutStrings */
 function filterOutStrings(values) {
   for (var i = values.length - 1; i >= 0; i--) {
-    if (typeof values[i] === 'string') {
-      values.splice(i, 1);
+    var newValues = values;
```

Comment:
This is still changing the original, remember the nature of reference data types. You are still changing the original array, you just put it into a different box.